config.py

```diff
@@ -63,7 +63,7 @@
 # with the names specified above
 # normally you should not have to touch this
 
-base_fname = '{subject}_' + study_name + '{extension}.fif'
+base_fname = '{subject}_' + study_name + '_{extension}.fif'
```

Comment:
the MNE convention wants it to be -extension from after raw data, i.e. mysubject-epo.fif